LSPS0/README.md

```diff
@@ -0,0 +1,603 @@
+# LSPS0 Transport Layer
+
+| Name    | `base_api`         |
+| ------- | ------------------ |
+| Version | 1                  |
+| Status  | For Implementation |
+
+## Status Specification
+
+All LSPS specifications, including this one, include a "Status" field.
+"Status" can be one of the following:
+
+* "For Implementation" - The specification has been widely reviewed by
+  LSPS participants, and is believed to have addressed all raised
+  issues, and LSPS participants recommend this specification to be
+  implemented.
+* "Stable" - The specification has been implemented by at least one
+  client and one LSP, which are developed by at least two different
+  organizations or open-source project teams, and both development
+  teams have reported interoperability without further modifications
+  or clarifications of the specification.
+
+## Motivation
+
+The transport layer describes how a client is able to request
+services from the LSP.
+
+### Actors
+
+'LSP' is the API provider, and acts as the server.
+'Client' is the API consumer.
```

Comment:
It is uppercase 'C' simply because it is the start of the sentence.

Could instead clarify as:

```Markdown
The 'LSP' is the API provider, and acts as the server.
The 'client' is the API consumer.
```